### ISSUE TITLE
Docs -  update terminology and product name

### ISSUE THREAD

**Issue by nearlythere** [drud/ddev#681]
## What happened (or feature request):

Product name changed and we cleared up some terminology. We should apply these changes to the documentation to clear up confusion. 

- Name of the product: DDEV Local
- Commands are lowercase: `ddev`
- Users configure a "Project" and not "SITE" or "APP." We refer to "your DDEV Local project."
- “DRUD Tech team” - The people behind DRUD. 
- DRUD is not a product, it is a technology company that makes products.

**Comment by nearlythere** [comment]
Checking this example:

> Make ddev executable: `chmod ugo+x ddev`

should this now be 

> Make DDEV Local executable: `chmod ugo+x ddev`

?

Also 

> Run `ddev` to test your installation. You should see ddev's command usage output.

should this be

> Run `ddev` to test your installation. You should see DDEV Local's command usage output.

**Comment by beeradb** [comment]
A few thoughts on this:

> Users configure a "Project" and not "SITE" or "APP." We refer to "your DDEV Local project."

I think any terminology like this should probably be thought through with ddev live in mind as well. ddev live has the additional complexity of having multiple environments (staging, dev, production, etc). Being able to be consistent across products would be beneficial.

> Commands are lowercase: ddev

We should also think through binary names. Eventually, we'll have CLIs for ddev live. Is that the same CLI? a difference CLI? if different, what should they both be named?

**Comment by nearlythere** [comment]
@beeradb Yeah 100% agreed. Seems like this isn't a simple find-and-replace, ha!

Also - I updated my comment above to make the other cases clear also. When we're writing about the commands as examples.

**Comment by horncologne** [comment]
For clarity, product names should be consistent and hyphenated:

- DDEV-Live
- DDEV-Local

(And DRUD should always be "DRUD", DRUD Tech, DRUD Technology LLC)

So ...

Case 1 - when you're talking about the commands: "Make DDEV-Local executable: `chmod ugo+x ddev`"

Case 2 - when you're talking about the commands: Run `ddev` to test your installation. You should see DDEV-Local's command usage output.

[Not 100% certain about the following]
Other cases ... If we consider DDEV-Local + -Live + future products/elements, DDEV is the overall platform, making it sensible to write where no context/clarification is needed:

"local DDEV-managed environment"
Using the DDEV command line interface (CLI)
DDEV Command Auto-Completion

**Comment by rfay** [comment]
So, one doesn't make a *product* executable, and one doesn't execute a product. You make a *file* (named 'ddev') executable and you run/execute the command 'ddev'. 

One also does not see the output of a *product* but of an execution event. 

This of course is the confusion of using two different cases for the product name and the command/file name.

**Comment by nearlythere** [comment]
@rfay @beeradb @horncologne I updated the original post with the latest thinking. When you have time, would you review and see if we are closer now?

**Comment by rfay** [comment]
Looks right to me, thanks!

**Comment by nearlythere** [comment]
@rfay Just checking - would you sanity check these four cases, A (ddev) or B (DDEV-Local)?

1. "Using Developer Tools with ddev" OR "Using Developer Tools with DDEV-Local"
2. Review the ddev issue queue OR Review the DDEV-Local issue queue
3. The ddev project is committed to OR The DDEV-Local project is committed to
4. "Note: ddev assumes that the files created by" OR "Note: DDEV Local assumes that the files created by"


I came up with this

1. B. Using Developer Tools with DDEV-Local
2. B. Review the DDEV-Local issue queue
3. B. The DDEV-Local project is committed to
4. A. Note: ddev assumes that the files created by

Does that look right to you?

**Comment by rfay** [comment]
For the first, it looks like maybe a title? We actually never have decided on NYT caps in titles or not. Maybe you'll get that settled! But I'd say "B: Using Developer Tools with DDEV-Local". Honestly, I'll come to terms with it :)

Second: "A. Review the ddev issue queue". Unless we rename it there. I would imaging the powers that be will be wanting to change the name and verbiage at the top of https://github.com/drud/ddev - in which case DDEV-Local will start to work.

Third: B. I imagine the "project" is DDEV-Local. So awkward.

Fourth: A: "Note: ddev assumes that the files created by". ddev is the thing that runs and does stuff. So if we're going to anthropomorphize something, it needs to be ddev. It's hard to anthropomorphize DDEV-Local.

Basically, when we're talking about the project or the product, DDEV-Local. When talking about the executable or the interaction with the tool, ddev. I know this contradicts the second one, but the fix for that is some edits on the top of the github project.

**Comment by nearlythere** [comment]
Thanks, @rfay! That helps. Do you think we can resolve the awkwardness? Do you think it's going to be an issue going forward, considering other products and services?

I agree the naming of the project on GitHub should be updated. The description of ddev should match also. 

@horncologne You want to check these cases too?

**Comment by rfay** [comment]
It's only an issue if I keep whining about it right? I"m sure I'll settle down :)

**Comment by rickmanelius** [comment]
This is a pain point that needs a discussion across stakeholders. This affects branding versus CLI usage (DDEV as the name and ddev as the command line usage) as well as how to effectively handle the namespacing with the multiple projects/products under DDEV... most notably how each would translate onto the CLI.

**Comment by rickmanelius** [comment]
Bleh. My comment was eaten when I closed a Chrome tab. @cyberswat, @sgrandchamp, and I met and reviewed the work to date (a huge thanks to everyone for consolidating this done to the key differentiators). At this point we've updated the issue summary and we should have reasonably accurate guidelines between referring to DDEV-Local the project/product and ddev the binary/command line. We still need to do a pass to square this all up in our docs, but we can now proceed.

**Comment by rfay** [comment]
Is there anything more to be done with this? If not, let's close it. Thanks!
